Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>The output of this is slightly weird when there is only one alert in the incident.</p><p>You might want to catch this case and either skip the timeline (which, in this case, shows a start alert event and some arbitrary end time).</p><p>Also having a title of "Incident duration" for the timeline might be better but it looks like this would need MP code changes (although we can do this before the 1.5.0 release)</p><p> </p><p>Having got connected to CyberSecuritySoc I still can't find an incident with &gt; 1 Alert so maybe that's a common case.</p><p> </p><p>Other thing is that with several of the incidents I get a weird None item on the graph. </p><p>Damn! Can't seem to repro this so maybe it's a weird edge case.</p><p> </p><p>Another thing (while trying to repro this None entity). It's kind of annoying to have to keep scrolling up and down to select a new incident and then to the run the cell. I wonder if you moved the setup code in this cell (perhaps put into functions) and then just call it with one or two lines of code. That way you can prob get the incident selector and the output in the same view - maybe.</p><p>Another change might be to have a check box with the Incident select to show or hide details. </p><p>I fiddled around with some code to do this from the NB but failed. Maybe we can add this to a 1.5.0 pre-release update.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>